tests/test_agent.py

```diff
@@ -40,7 +40,20 @@ def test_custom_metrics(mock_send_report, handler_with_events, mock_context):
     iopipe, handler = handler_with_events
     handler(None, mock_context)
 
+    assert len(iopipe.report.custom_metrics) == 7
+    # Decimals are converted to strings
+    assert iopipe.report.custom_metrics[6]['s'] == '12.300000000000000710542735760100185871124267578125'
```

Comment:
Does the node agent cast to floats? Maybe we should make this behavior consistent.